src/core/identifiers/identifiers.test.ts

```diff
@@ -1,28 +1,28 @@
-import { validateDSNPMessageId } from "./identifiers";
+import { validateDSNPAnnouncementId } from "./identifiers";
 
-describe("validateDSNPMessageId", () => {
-  const validDSNPMessageIds = [
+describe("validatedsnpAnnouncementId", () => {
+  const validdsnpAnnouncementIds = [
     "dsnp://0123456789ABCDEF/0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF", // Uppercase
     "dsnp://0123456789abcdef/0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef", // Lowercase
   ];
 
-  const invalidDSNPMessageIds = [
+  const invaliddsnpAnnouncementIds = [
     "dsnp://badbadbad/0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF", // Bad user id
     "dsnp://0123456789ABCDEF/badbadbad", // Bad message id
     "dsnp://0123456789ABCDE/0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF", // User id too short
     "dsnp://0123456789ABCDEFA/0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF", // User id too long
     "dsnp://0123456789ABCDEF/0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDE", // Message id too short
   ];
 
-  for (const id of validDSNPMessageIds) {
+  for (const id of validdsnpAnnouncementIds) {
```

Comment:
did we downcase `DSNP` here intentionally?

Reply:
Nope. Will Fix.